src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -101,7 +101,7 @@ private static void CheckForBadShared(List<string> arguments)
 
         private static void ReferenceNetstandardDllIfCoreClr(TempDirectory currentDirectory, List<string> arguments)
         {
-#if NETCOREAPP2_0
+#if NETCOREAPP2_1
```

Comment:
Would it be better to use `!NET46`? We will likely update NetCoreApp target more often than Desktop FX.

Reply:
Yes. I will change.